### PR TITLE
Include missing default GPO setting values in Windows-Time-Service-Tools-and-Settings.md

### PR DESCRIPTION
**Pull request by sm-msft**
Expand the GPO setting list on this doc page to include missing entries. 

For the full list, open "Local Group Policy Editor" on your machine locally, navigate to "Computer Configuration" -> "Administrative Templates" -> "System" -> "Windows Time Service" and click on "Global Configuration Settings" in the right-side pane. This opens a list of GPO settings in a new page. Select "Enabled" to have the default values for each setting to be populated on this page. 

***Important*** - Click "Cancel" once you are done looking through the settings, else you may end up applying them on your machine locally.

### CODE REVIEW

**Comment by Court72** [comment]
@robinharwood, @Xelu86

Can you review the proposed changes? 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: WindowsServerDocs/networking/windows-time-service/Windows-Time-Service-Tools-and-Settings.md [hunk 1]
```diff
@@ -261,6 +261,12 @@ Below you'll find the default values for the **Global Configuration Settings** o
 |Group Policy setting|Default value|
 | --- | --- |
 |AnnounceFlags|10|
+|ChainDisable|0|
+|ChainEntryTimeout|16|
+|ChainLoggingRate|30|
```
Comment (mitschmi) [inline WindowsServerDocs/networking/windows-time-service/Windows-Time-Service-Tools-and-Settings.md:266]: Should we also have "ChainMaxEntries" with a default of 128:
image

**Review by mitschmi — CHANGES_REQUESTED** [review]

File: WindowsServerDocs/networking/windows-time-service/Windows-Time-Service-Tools-and-Settings.md (hunk 1 above)
Comment (sm-msft) [reply WindowsServerDocs/networking/windows-time-service/Windows-Time-Service-Tools-and-Settings.md:266]: Thanks!

**Review by mitschmi — APPROVED** [review]

**Comment by sm-msft** [comment]
@robinharwood Please review/signoff on these changes.

#label:"sign-off"

**Pull request merged**